src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Represents a deadline task.
+ */
+public class Deadline extends Task {
+    private LocalDate date;
+    private Date time = null;
+    SimpleDateFormat timeFormatter = new SimpleDateFormat("h:mm a");
+    private DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("MMM d yyyy");
+
+    /**
+     * Creates an instance of a deadline task.
+     * @param description  Contents of the deadline task in the format "deadline XXX /by yyyy-mm-dd HH:mm".
+     * @throws DukeException  If format of date and/or time given is not yyyy-mm-dd HH:mm.
+     */
```

Comment:
Should there be a line separating description and parameter section for JavaDoc?